app/assets/javascripts/discourse/app/components/reviewable-item.hbs

```diff
@@ -68,12 +68,7 @@
     {{/if}}
   </div>
 
-  {{#if
-    (or
-      (eq this.reviewable.type "ReviewableFlaggedPost")
-      (eq this.reviewable.type "ReviewableChatMessage")
```

Comment:
This is a plugin reviewable type and should not be leaking into core.

Reply:
This I don't know how to fix then, sorry.